cmd/harmony/main.go

```diff
@@ -416,6 +425,27 @@ func setupConsensusAndNode(nodeConfig *nodeconfig.ConfigType) *node.Node {
 	return currentNode
 }
 
+func setupBlacklist() (*map[ethCommon.Address]bool, error) {
+	if _, err := os.Stat(*blacklistPath); os.IsNotExist(err) {
+		return nil, errors.New(fmt.Sprintf("blacklist file not found at `%s`", *blacklistPath))
+	}
```

Comment:
No point to check file and then read. Just read and if no file then error will happen

Reply:
Ah right, will change